src/site/content/en/fast/your-first-performance-budget/index.md

```diff
@@ -298,9 +298,9 @@ When you try to build a great site, **set Lighthouse performance score budget to
 
 ## Prioritize
 
-Ask yourself what level of interaction you expect on your site. If it's a news website, users' primary goal is to read content so you should focus on rendering quickly and keeping FCP low. Doggos.com visitors want to click on relevant links as soon as possible, so the top priority is low TTI.
+Ask yourself what level of interaction you expect on your site. If it's a news website, users primary goal is to read content so you should focus on rendering quickly and keeping FCP low. Doggos.io visitors want to click on relevant links as soon as possible, so the top priority is low TTI.
```

Comment:
I can't get Doggos.io to load. Doggos.com loads immediately.

Reply:
It's not an example domain. It looks like 'Doggos.com' needs to be used everywhere instead of changing this to Doggos.io.

cc: @mihajlija

Reply:
Changes all references of **doggos.io/Doggos.io** domain with **"Doggos.com"** as per suggestions.